### DISCUSSION TITLE
School Issued email doesn't get verified

### DISCUSSION THREAD

**Discussion by Sazid99246** [community/community#53069] [GitHub Education]
Hi, everyone. I am from the Green University of Bangladesh and my university gave me an email address. The domain of the email given by my university is **student.green.ac.bd**. When I want to apply to the GitHub student developer pack with my email and my institution is of mine, I get this kind of message: 
**We require applicants of Green University of Bangladesh to use one of these school-issued email addresses to apply:
- cse.green.edu.bd
- green.ac.bd
- gub.ac.bd
- student.green.edu.bd
-
Please select a different email address.**

But my university right now is not providing any of the domains. It just gives us the domain **student.green.ac.bd**.

And when I want to enroll in GitHub education, it tells me: **We require applicants to use a school-issued email address to apply.**

I think this is a bug in your database. Please add the domain **student.green.ac.bd** to your database for the students of the Green University of Bangladesh. Thanks in advance.

**Comment by Enissays** [comment]
I have the same problem here, with a Moroccan university 
"to use one of these school-issued email addresses to apply:
- taalim.ma" but my school doesn't provide a mail with this domain, and isn't even affiliated to taalim, kinda weird that this happens ?

**Reply by Sazid99246** [reply]
And they don't even reply to this kind of thread I have ever seen🙃

**Reply by hasnainahmed007** [reply]
I have same problem, from green university of bangladesh.

**Comment by samus-aran** [comment]
Hi @Sazid99246 & @Enissays  👋

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "Other". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

Hope this helps! Let us know how you get along 👍

**Reply by liamsimfty** [reply]
Hi Samus i've alreay make a ticket 
Ticket ID: 3090216

**Comment by romdotdog** [comment]
I figured it out. There's a shortcut on the first page where it attempts to link you with your school and it removes the "Email" field from where you're supposed to fill it out. It's pre-filled with your first email which is often not your school email. Turns out, if you try to take the shortcut, it attempts to match it with the WRONG email. That's why it rejects you. Don't take the shortcut.

**Comment by Manvanth-Gowda-M** [comment]
bro my college don't even give us a school email , only with id we can verify it is there any solution can anyone plese help with it
